---
title:
author:
header-includes:
  - \usepackage{color}
  - \definecolor{TitleColor}{RGB}{30,30,30}
  - \usepackage{caption}
  - \captionsetup[figure]{labelformat=empty}
output:
  pdf_document:
    latex_engine: xelatex
    fig_caption: yes
mainfont: Times New Roman
fontsize: 12pt
---

<center>

![
**Figure S1. Experimental rice plots established in Center for Ecological Research, Kyoto Univeristy (34$^{\circ}$ 58'18'' N, 135$^{\circ}$ 57'33'' E) in 2017.** **(a)** An image of the experimental rice plot (90 cm $\times$ 90 cm; the photo was taken by M. Ushio). **(b)** Three rice individuals were grown in each Wagner pot. Growth rates of four individuals at the center of each plot were monitored for agricultural purposes. The growth data are being analyzed for different purposes and thus are not shown in this study. **(c)** Five experimental rice plots were established in an experimental field in Center for Ecological Research, Kyoto University (the image taken from Google Earth). Red squares indicate the locations of the rice plots.
](../00_ReformatFigs/SI_Fig_01.pdf)

</center>

<center>

![
**Figure S2. Rarefaction curve and results of negative and positive controls.** Rarefaction curve (**(a)**-**(d)**) and the results of analysis of negative (**(e)**-**(h)**) and positive (**(i)**-**(l)**) controls. Most of the rarefaction curves reached a plateau, suggesting that the sequencing captured most diversity of the target taxa (**(a)**-**(d)**). Investigations of negative control samples suggested that no serious contamination occurred during sample transportation, eDNA extraction or library preparation (Almost no non-standard DNAs were detected from Standard NC, Field NC or PCR NC; **(e)**-**(h)**). No dramatic reductions in DNA copy numbers of the top 10 dominant taxa were observed in samples from which DNAs were extracted before the storage duration exceeded 3 months (red points in **(i)**-**(l)**; note that DNAs of all samples analyzed in the time series analysis were extracted within 1-2 months after sampling). The box-plot elements are defined as follows: center line, median; box limits, upper and lower quantiles; whiskers, 1.5 $\times$ interquantile range; points, outliers. Together, these results suggest that the water sampling and DNA analysis in the present study enabled reliable illustrations of the dynamics of ecological communities in the rice plots. Detailed descriptions of the negative and positive control samples are provided in the Supplementary Text.
](../00_ReformatFigs/SI_Fig_02.pdf)

</center>



<center>

![
**Figure S3. Validation of the quantitative MiSeq sequenicing ("qMiSeq").** **(a)** Examples of the relationship between standard DNA copy numbers and sequence reads. Slope of the linear regression changes among samples, partly because of the different intensity of PCR inhibition. The regression lines with maximum, medium, and minimum slope values are shown by red, dark red, and black points and lines as examples. **(b)** Distributions of $R^2$ values of the linear regression (610 samples $\times$ 4 marker regions). Most samples and regions show $R^2 > 0.9$, suggesting that the sequence reads for each standard DNA are proportional to the initial DNA copy numbers. **(c)** The relationship between total DNA concentrations quantified by fluorescent-based method and the total DNA copy numbers estimated by qMiSeq. **(d)** The relationship between total 16S copy numbers measured by quantitative PCR (qPCR) and qMiSeq. **(e)** The relationship between total 16S copy numbers measured by qPCR and 16S sequence reads. Dashed line and colors indicate a linear regression and the rice plots, respectively. **(f)**-**(i)** Comparisons of 16S amplicon sequencing (qMiSeq) and 16S extracted from the results of shotgun metagenomic analysis by the phyloFlash pipeline. Barplots indicate community composition revealed by the two methods. The number of taxa detected by each method is shown at the top of the barplots. Because the shotgun metagenomic analysis requires much deeper sequencing than an amplicon sequencing, only four samples were analyzed for the comparison. Samples were from **(f)** Plot 1 on 2017/7/14, **(g)** Plot 1 on 2017/7/15, **(h)** Plot 5 on 2017/7/14, and **(i)** Plot 5 on 2017/7/15. Detailed methods and discussions are presented in Supplementary Information.
](../00_ReformatFigs/SI_Fig_03.pdf){width=70%}

</center>


<center>

![
**Figure S4. Patterns in DNA time series of major taxonomic groups in the study.** Temporal patterns of ASV diversity, major taxa and the DNA copy numbers for **(a)**-**(b)** prokaryotes, **(c)**-**(d)** fungi, **(e)**-**(f)** metazoans, and **(g)**-**(h)** viridiplantae.
](../00_ReformatFigs/SI_Fig_04.pdf)

</center>


<center>

![
**Figure S5. Time-varying properties of the interaction network and dependence of dynamic stability on the community size.** **(a)** Changes in the network structures. Interaction networks in Plot 1 on 1 July, 2 August and 18 September 2017 are shown as examples of time-varying interaction networks. Blue and red arrows indicate positive and negative causal interactions, respectively. **(b)** The number of interactions that each ASV had on each day. Note that $y$-axis is log-transformed. **(c)** Dynamic stability of the ecological community. Different colors, lines and symbols indicate different plots. **(d)**-**(g)** Dependence of dynamic stability on the community size. The scatter plots comparing the dynamic stability of the full community (1197 ASVs) and the most dominant 800 (**(d)**), 400 (**(e)**), 200 (**(f)**), and 100 (**(g)**) ASVs are shown. Dashed diagonal line indicates 1:1 line. Dashed horizontal and vertical lines indicate the dynamic stability value 1 (values lower than 1 indicate locally stable dynamics). As shown in **(c)**-**(f)**, the removals of moderately abundant ASVs stabilize the community dynamics, suggesting that fluctuations in the abundance of the moderately abundant ASVs could contribute to the unstable community dynamics in the rice plots.
](../00_ReformatFigs/SI_Fig_05.pdf){width=80%}

</center>


<center>

![
**Figure S6. Additional patterns in network properties.** **(a)** Mean air temperature and ASV diversity, **(b)** total DNA copy numbers and ASV diversity, **(c)** Mean air temperature and total DNA copy numbers, **(d)** The number of interactions and ASV diversity. **(e)** ASV diversity and dynamic stability and **(f)** Dynamic stability and coefficients of variation of community dynamics. Red lines indicate statistically clear nonlinear regressions by the general additive model (GAM). Gray shaded region indicates 95% confidence interval of GAM.
](../00_ReformatFigs/SI_Fig_06.pdf){width=80%}

</center>


<center>

![
**Figure S7. Comparisons between observed network properties and those of randomly shuffled time series.** **(a)** Mean interaction strength and ASV diversity, **(b)**, **(c)** Mean interaction capacity, mean air temperature and total DNA copy numbers, and **(d)** mean interaction capacity and ASV diversity, and **(e)**, **(f)** connectance, mean air temperature and total DNA copy numbers. Gray points indicate patterns generated by analyzing randomly shuffled time series. Light blue points and line indicate patterns for the original time series. The observed patterns were not reproduced by the analyses of the randomly shuffled time series.
](../00_ReformatFigs/SI_Fig_07.pdf)

</center>


<center>

![
**Figure S8. Species-level patterns of interaction strength, interaction capacity, and connectance.** **(a)** Relationships between the number of interactions and mean interaction strength per link. **(b)** Relationship between species-level interaction capacity and phylum. Species-level interaction capacity can be calculated for each day in each plot, and the median value of the interaction capacity is plotted as a representative interaction capacity for each ASV in **(b)**. The box-plot elements are defined as follows: center line, median; box limits, upper and lower quantiles; whiskers, 1.5 $\times$ interquantile range; points, outliers. Dashed lines indicate the median interaction capacity for Bacteria and Eukaryota. Bacterial taxa generally show a higher interaction capacity than eukaryotes, suggesting that the potential maximum diversity of Bacteria is higher than that of Eukaryota. **(c)** The distribution of species-level connectance. Dashed vertical lines indicate median values of species-level connectance for each superkingdom. Colors indicate superkingdom. **(d)** The relationship between an index of body size (= 1/total DNA copy numbers) and interaction capacity. Archaea and Undetermined are excluded due to the small number of data points. There is a negative relationship between the body size index and interaction capacity for Bacteria and Eukaryota, suggesting that more abundance, smaller organism (e.g, prokaryotes) could have higher interaction capacity. A higher diversity in smaller organisms might be related to the potential higher interaction capacity of the smaller organisms. **(e)** An enlarged figure of **(d)**. Only regression lines are shown to clarify the pattern.
](../00_ReformatFigs/SI_Fig_08.pdf){width=100%}

</center>

<center>

![
**Figure S9. Causal relationships among the network properties.** A cell filled with gray color indicates that there is a causal influence from a column variable ("Causal property") to a row variable ("Effect property"), which is detected by convergent cross mapping. Community diversity is influenced by interaction capacity and interaction strength. Also, dynamic stability is influenced by community diversity, which is in agreement with Ushio et al. (2018) https://doi.org/10.1038/nature25504. On the other hand, "Weak interaction index" does not influence dynamic stability, but influence "Mean C.V." (i.e., realized population fluctuations). 
](../00_ReformatFigs/SI_Fig_09.pdf)

</center>



<center>

![
**Figure S10. Empirical evidence supporting the interaction capacity hypothesis in other systems.** **(a)**-**(f)** Predictions of community diversity by nonlinear regression using temperature and abundance. Predicted and observed diversity for metagenomic data generated by Tara Oceans research expedition (**(a)**), global soil microbial community (**(b)**), a fish community in a marine costal region in Kyoto, Japan (**(c)**), prokaryotic communities in Japanese lakes (**(d)**), zooplankton communities in Lake Suwa, Japan (**(e)**) and macroinvertebrate communities in freshwater tributary lagoons surrounding Lake Biwa, Japan (**(f)**). Values at the bottom right in panels indicate adjusted $R^2$ of GAM. Gray shaded region indicates 95% confidence interval of the linear regression between predicted and observed diversities. **(g)**-**(h)** The negative relationship between interaction strength and community diversity. Examples from experimental microbial communities (**g**, Ratzke et al. 2020) and ant-plant interactions under field conditions (**h**, Yamawo et al. 2021).
](../00_ReformatFigs/SI_Fig_10.pdf)

</center>




\newpage
**Table S1. Statistical results of GAM between network properties.**

<center>

![
](../00_ReformatFigs/SI_Table_01.pdf){width=100%}

</center>
